src/test/java/com/iota/iri/service/snapshot/impl/SnapshotProviderImplTest.java

```diff
@@ -11,14 +16,30 @@
 import com.iota.iri.service.snapshot.SnapshotException;
 import com.iota.iri.service.spentaddresses.SpentAddressesException;
 
+import java.util.HashMap;
+
 public class SnapshotProviderImplTest {
```

Comment:
You want to help add code coverage by writing a unit test to `PersistSnapshot` here?

Reply:
@DyrellC ?

Reply:
Added